pixi.toml

```diff
@@ -15,7 +15,7 @@ chameleon = "*"
 defusedxml = "*"
 email_validator = "*"
 execnet = "*"
-fastkml = ">=0.11"
+fastkml = "<1"
```

Comment:
I think I would prefer a version string <1.0.0

Reply:
There is no difference between <1 and <1.0.0.